### DISCUSSION TITLE
enhance playlist shuffle ui

### DISCUSSION THREAD

**Discussion by wobie** [navidrome/navidrome#2397] [Ideas]
wondering if possible to separate shuffle button from repeat modes, and a feature to set default now playing mode to shuffle.

**Comment by deluan** [comment]
This is a limitation of the current audio player, and it is a deprecated project. When we replace it with our own player, this will be taken in consideration.

**Reply by bilogic** [reply]
How about make `Repeat` the default rather than `In order`?

I normally just hit play... until the music stops, then I have to go fiddle with Navidrome again... happens everyday :)
